crates/astria-auctioneer/src/metrics.rs

```diff
@@ -1,25 +1,153 @@
-use telemetry::{
+use astria_telemetry::{
     metric_names,
     metrics::{
         self,
+        Counter,
+        Gauge,
+        IntoF64,
         RegisteringBuilder,
     },
 };
 
-pub struct Metrics {}
+const AUCTION_BIDS_PROCESSED_LABEL: &str = "auction_bids_processed";
+const AUCTION_BIDS_PROCESSED_ADMITTED: &str = "admitted";
```

Comment:
i feel like received is a better word than admitted. there is no entry barrier for the bids to gain admission, received sounds like a better word to me at least.

Reply:
The distinction is important actually (received is processed + dropped), but I did change the nomenclature in 27af6c11 (now the histogram is just "auction_bids" with labels ["processed", "dropped"]).